File: Common/src/test/java/com/jiangli/common/test/HashTest.java
package com.jiangli.common.test;

import org.junit.Test;

/**
 * @author Jiangli
 * @date 2018/6/27 16:49
 */
public class HashTest {
    @Test
    public void test_() {
        Object o = new Object();
        System.out.println(System.identityHashCode(o));
        System.out.println(o.hashCode());
    }



}

File: Doc/src/main/java/com/jiangli/doc/txt/importdata/HashTest.kt
package com.jiangli.doc.txt.importdata

import com.jiangli.common.utils.YufaCodis
import com.jiangli.doc.sql.helper.zhihuishu.Zhsutil
import org.springframework.jdbc.core.ColumnMapRowMapper

/**
 *
 *
 * @author Jiangli
 * @date 2018/5/29 15:50
 */

fun main(args: Array<String>) {
    val userId = Zhsutil.convertUUID("VvM1r7zX")
    println(userId)

    val CURRENT_ENV = Env.YUFA
    val configMap = getConfig()
    val targetDB = configMap[CURRENT_ENV]!!.jdbc

    val query = targetDB.query("select ID from db_teacher_home.TH_TEACHER where USER_ID = $userId", ColumnMapRowMapper())
    if (query.size == 1){
        val thId = query[0]["ID"] as Int
        println(thId)

        val codis = YufaCodis()
        codis.execute("del th:teacher:$thId")
    }

}